repos/system_upgrade/el7toel8/actors/quaggareport/actor.py

```diff
@@ -0,0 +1,44 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import QuaggaToFrrFacts, Report
+from leapp.reporting import create_report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+COMMON_REPORT_TAGS = [
+    reporting.Tags.NETWORK,
+    reporting.Tags.SERVICES
+]
+
+
+class QuaggaReport(Actor):
+    """
+    Checking for babeld on RHEL-7.
+
+    This actor is supposed to report that babeld was used on RHEL-7
+    and it is no longer available in RHEL-8.
+    """
+
+    name = 'quagga_report'
+    consumes = (QuaggaToFrrFacts, )
+    produces = (Report, )
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        try:
+            quagga_facts = next(self.consume(QuaggaToFrrFacts))
+        except StopIteration:
+            return
+        if 'babeld' in quagga_facts.active_daemons or 'babeld' in quagga_facts.enabled_daemons:
+            create_report([
+                reporting.Title('Babeld is not available in FRR'),
```

Comment:
Can we provide a little bit detail about FRR here via link `reporting.ExternalLink(url='https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/setting-your-routing-protocols_configuring-and-managing-networking', title='Setting routing protocols in RHEL8')`

Reply:
it has been addressed